### PR TITLE
update version (#672)

### PR DESCRIPTION
**Pull request by zerobfd**

### CODE REVIEW

File: scripts/asm-installer/asm_vm [hunk 1]
```diff
@@ -23,9 +23,9 @@ _CI_ISTIOCTL_REL_PATH="${_CI_ISTIOCTL_REL_PATH:=}"
 ### Internal variables ###
 MAJOR="${MAJOR:=1}"; readonly MAJOR;
 MINOR="${MINOR:=9}"; readonly MINOR;
-POINT="${POINT:=3}"; readonly POINT;
+POINT="${POINT:=5}"; readonly POINT;
 REV="${REV:=2}"; readonly REV;
-CONFIG_VER="${CONFIG_VER:="1-staging"}"; readonly CONFIG_VER;
+CONFIG_VER="${CONFIG_VER:="1-unstable"}"; readonly CONFIG_VER;
```
Comment (jasonwzm) [inline scripts/asm-installer/asm_vm:28]: This should be a config version right?

File: scripts/asm-installer/install_asm [hunk 2]
```diff
@@ -33,7 +33,7 @@ _CI_I_AM_A_TEST_ROBOT="${_CI_I_AM_A_TEST_ROBOT:=0}"
 ### Internal variables ###
 MAJOR="${MAJOR:=1}"; readonly MAJOR;
 MINOR="${MINOR:=9}"; readonly MINOR;
-POINT="${POINT:=3}"; readonly POINT;
+POINT="${POINT:=5}"; readonly POINT;
 REV="${REV:=2}"; readonly REV;
 CONFIG_VER="${CONFIG_VER:="1-staging"}"; readonly CONFIG_VER;
```
Comment (jasonwzm) [inline scripts/asm-installer/install_asm:38]: Config version here.

**Review by jasonwzm — CHANGES_REQUESTED** [review]

File: scripts/asm-installer/asm_vm (hunk 1 above)
Comment (zerobfd) [reply scripts/asm-installer/asm_vm:28]: Yes, this PR is just to kick the tests off. The actual release commit will be done via CLI to take care of these.

Comment (jasonwzm) [reply scripts/asm-installer/asm_vm:28]: Oh I see...

**Pull request merged**